R/truncate_triangles.R

```diff
@@ -8,9 +8,9 @@
 #'
 #' @param n Integer indicating the number of retrospective
 #'   truncated triangles to be generated, always starting from the most
-#'   recent reference time. Default is to generate truncated matrices
-#'   for all triangles that would have a sufficient number of rows to generate
-#'   a nowcast from.
+#'   recent reference time. Default is to generate truncated matrices for
+#'   the maximum delay (number of columns - 1) or one less than the number of
```

Comment:
no indication of which it will pick. I would have thought that most users would expect it to do one for each row so perhaps we need some rational for this default as well

Reply:
Yes aI think this is preferable but not really my point here you don't say it uses the min

Reply:
Will edit